requirements.txt

```diff
@@ -0,0 +1 @@
+bokeh==0.12.7
```

Comment:
@arkostyuk I linked an example to make you spot the comments on top ;)
Add a line a above, like `# web_widget_bokeh_chart`